src-react/components/dashboard/AddonInput.jsx

```diff
@@ -24,39 +24,25 @@ export class AddonInput extends Component {
 
   onSubmit (e) {
     const { url } = this.state
-
     if (url !== '' && checkURL(url)) {
       this.props.onSubmit(url)
+      document.getElementById('addonInput').value = ''
```

Comment:
This might cause unforeseeable problems due to html mutations not matching the react state. Do something like this ```this.setState({url: ''})``` this will reRender the AddonInput to be empty. Also I think its VERY important you go back and review react and its work flow. setState should be one of your goto tools for changing values on the UI

Reply:
reverted